estudios_socioeconomicos/views.py

```diff
@@ -1,3 +1,33 @@
-# from django.shortcuts import render
+from django.views.generic import ListView
+from django.contrib.auth.models import User
+from django.contrib.auth.decorators import login_required, user_passes_test
+from django.utils.decorators import method_decorator
 
-# Create your views here.
+
+from perfiles_usuario.utils import is_administrador
+from .models import Estudio
+
+decorators = [user_passes_test(User, is_administrador), login_required]
+
+
+@method_decorator(decorators, name='dispatch')
```

Comment:
I think this is hard to understand and it's preferred to have both decorators instead:
@login_required
@user_passes_test(is_administrador)